### ISSUE TITLE
Random string

### ISSUE THREAD

**Issue by meiremans** [aviaviavi/curl-runnings#35]
Hi

Is there a way to generate a random string and use that string during the whole test run?

with kind regards

**Comment by aviaviavi** [comment]
Hi there!

Since curl runnings loads environment variables at the beginning of the test suite, using the `${RANDOM}` environment variable will generate a random number, which you can use as a string in your tests. Will that work for you?

**Comment by aviaviavi** [comment]
You can also generate your own random strings and set them as environment variables that you reference in your curl-runnings tests. Nothing is built into the spec that has random functionality, but one could certainly be added if there's a good use case

**Comment by aviaviavi** [comment]
Got it. So CR won't evaluate arbitrary bash expressions which is why `RANDOM | tr '[0-9]' '[a-zA-Z]'` wasn't working. Just environment variables as they are. So yeah for now, just RANDOM, or simply invoke curl runnings with something like `USER_ID="$(RANDOM | tr '[0-9]' '[a-zA-Z]')" curl-runnings -f <your_file>`

Sounds like having some random string + number generators natively supported by curl-runnings would be useful though, I do agree about that.

**Comment by meiremans** [comment]
Thank you very much for your quick reply. 
I agree a random string and number generator would be useful. but the current workaround works fine for me.

On an other note, I am running this inside a docker container. Would it be useful if I push my dockerfile for other docker users?

**Comment by aviaviavi** [comment]
absolutely! please open a PR whenever you have the chance, thanks so much!
